src/test/cypress/cypress.config.ts

```diff
@@ -81,7 +81,7 @@ export default defineConfig({
         },
         specPattern: ['init/ImportUsers.cy.ts', 'e2e/**/*.cy.ts'],
         supportFile: 'support/index.ts',
-        baseUrl: 'http://localhost:8080',
+        baseUrl: 'http://localhost:9000',
```

Comment:
I don't think you should change this

Reply:
oh yeah during the huge merge conflict resolving, I overlooked some pushed configuration files 😓 
Undid those changes.